src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -40,6 +40,9 @@
         "symfony/translation": "~2.0,>=2.0.5|~3.0.0",
         "symfony/var-dumper": "~2.6|~3.0.0"
     },
+    "conflict": {
+        "symfony/config": "<2.7"
```

Comment:
The require-dev does not work if symfony/config is followed as a transitive dependency during dep=... builds. So `conflict` avoids making `symfony/config` a hard requirement, but makes sure we get at least 2.7 *if* we get it.